app/controllers/welcome_controller.rb

```diff
@@ -0,0 +1,5 @@
+class WelcomeController < ApplicationController
```

Comment:
Do you need this controller for anything?

Reply:
No, removed it